pkg/agent/plugin/workloadattestor/windows/windows_windows_test.go

```diff
@@ -42,6 +44,10 @@ var (
 )
 
 func TestAttest(t *testing.T) {
```

Comment:
Can you add a TestConfigure(t *testing.T)? I think it will only have 2 test cases: `sucess to decode configuration`, `failed to decode`